src/main/java/org/example/objects/User.java

```diff
@@ -1,30 +1,39 @@
 package org.example.objects;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import org.example.fuzzy_matcher.domain.Document;
 import org.example.fuzzy_matcher.domain.Element;
 import org.example.fuzzy_matcher.domain.ElementType;
 
 import java.util.List;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class User {
   private int id;
```

Comment:
I have a question, is the id user in this attribute loaded from the json?

Reply:
if the input JSON contains additional properties that are not defined in the User class, Jackson will simply ignore them